### PR TITLE
Fix vet issues in model

### PR DESCRIPTION
**Pull request by noxer**
#### Summary
This PR fixes all issues found by the new `mattermost-govet` in the `model` package.

#### Ticket Link
Loosely associated with https://mattermost.atlassian.net/browse/MM-45991 and https://mattermost.atlassian.net/browse/MM-46238

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: model/plugin_valid.go [hunk 1]
```diff
@@ -15,7 +15,8 @@ const (
 )
 
 // ValidId constrains the set of valid plugin identifiers:
-//  ^[a-zA-Z0-9-_\.]+
+//
```
Comment (isacikgoz) [inline model/plugin_valid.go:18]: Why there is an empty line needed here?

**Review by isacikgoz — APPROVED** [review]
LGTM, just a question out of curiosity.

File: model/plugin_valid.go (hunk 1 above)
Comment (noxer) [reply model/plugin_valid.go:18]: This empty line is required for the new comment formatting of Go 1.19. The new `gofmt` now automatically formats code snippets with an empty line before and after them.

**Review by agnivade — APPROVED** [review]
Looks good

**Pull request merged**